### PR TITLE
Fix paste-to-upload permissions and Edge support

### PR DESCRIPTION
**Pull request by BenLubar**

### CODE REVIEW

**Comment by julianlam** [comment]
LGTM. Awaiting signoff from @barisusakli

**Comment by BenLubar** [comment]
I misremembered which browser had the problem with file.name. Firefox is still broken, but at least Edge works.

File: static/lib/composer/uploads.js [hunk 1]
```diff
@@ -257,6 +258,10 @@ define('composer/uploads', [
 			uploads.inProgress[post_uuid] = uploads.inProgress[post_uuid] || [];
 			uploads.inProgress[post_uuid].push(1);
 
+			if (params.formData) {
+				params.formData.append('cid', cid);
```
Comment (barisusakli) [inline static/lib/composer/uploads.js]: Do we need this? cid is already passed in the data below.

Comment (barisusakli) [inline static/lib/composer/uploads.js]: Nevermind saw the other issue.

**Pull request merged**